CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Added support for Bitchx (via @troywilson_)
 - Added support for EditorConfig (via @chadluo)
 - Add com.agilebits.onepassword4.plist in 1Password (via @amatos)
+- Added support for Versions (via @amatos)
```

Comment:
Can you add it to the README? See https://github.com/lra/mackup/blob/master/CONTRIBUTING.md